cluster-autoscaler/cloudprovider/rancher/examples/config.yaml

```diff
@@ -1,9 +1,13 @@
-# rancher server credentials
+# When you create a token in the Rancher UI, 
+# it shows you the API endpoint, that is the Rancher URL with the /v3 appended at the end.
+# Make sure the URL does not contains this /v3 appended.
+# Rancher Base URL (Not API endpoint)
```

Comment:
@diegonayalazo, It would be good if we could write the comment like this: (Rancher Base URL (Not API endpoint) should come first). 

```
# Rancher Base URL (Not API endpoint)
# When you create a token in the Rancher UI, 
# it shows you the API endpoint, that is the Rancher URL with the /v3 appended at the end.
# Make sure the URL does not contain this /v3 appended.
```